cli/ballerina-tool/src/main/java/org/ballerinalang/tool/util/BFileUtil.java

```diff
@@ -118,11 +118,18 @@ public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOEx
      *
      * @param orgName     Org name
      * @param packageName Package name
+     * @param version Package version
      * @param className   Class name
      * @return Qualified class name
      */
-    public static String getQualifiedClassName(String orgName, String packageName, String className) {
+    public static String getQualifiedClassName(String orgName, String packageName, String version,
+                                               String className) {
+
+
         if (!Names.DEFAULT_PACKAGE.value.equals(packageName)) {
+            if (!"".equals(version)) {
+                className = version.replace('.', '_') + "." + className;
```

Comment:
looks like version.replace('.', '_') is a common code seen on multiple occurrences in this PR that can be moved to a method ?

Reply:
even it is multiple occurred, these are in different modules and packages. Then moving to method will not have any advantage and this is also inside util method of each module